### ISSUE TITLE
Copying toolbox results in error. 

### ISSUE THREAD

**Issue by technotherion** [2Shirt/WizardKit#1]
Upon using the Copy Toolbox script on Windows 7 

```
Initializing...

Copying .bin...

Path not found - C:\!_v:~0,-1!\Tools

Copying .cbin...

Path not found - C:\!_v:~0,-1!\Tools

Copying Main Toolbox...

Extracting Ninite installers...

Aborted.

Press any key to exit...
```

**Comment by technotherion** [comment]
Fixed in [patch 3](https://github.com/2Shirt/WizardKit/compare/master...technotherion:patch-3)

**Comment by 2Shirt** [comment]
Issue fixed in 6fc0426
